jsplot/src/test/java/tech/tablesaw/examples/HoverBroadcastExample.java

```diff
@@ -0,0 +1,64 @@
+package tech.tablesaw.examples;
+
+import java.util.Random;
+import java.time.LocalDateTime;
+
+import tech.tablesaw.api.DateTimeColumn;
+import tech.tablesaw.api.DoubleColumn;
+import tech.tablesaw.api.IntColumn;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.traces.ScatterTrace;
+import tech.tablesaw.plotly.components.Axis;
+import tech.tablesaw.plotly.components.Layout;  
+import tech.tablesaw.plotly.components.Figure;
+import tech.tablesaw.plotly.event.HoverBroadcastBody;
+import tech.tablesaw.plotly.event.HoverEventHandler;
+
+public class HoverBroadcastExample {
+
+    public static void main(String[] args) throws Exception {
+	int seriesLen = 50;
+    	Random rng = new Random();
+    	Double[] prices = new Double[seriesLen];
+    	int[] volumes = new int[seriesLen];
+    	LocalDateTime startDT = LocalDateTime.of(2019, 1, 1, 9, 30, 0);
+    	LocalDateTime[] times = new LocalDateTime[seriesLen];
+    	  
+    	HoverBroadcastBody hbb = HoverBroadcastBody.builder()
+        .subPlots( new String[] {"xy", "xy2"} )
+        .numTraces(2)
+        .build();
+        
+    	HoverEventHandler heh = HoverEventHandler.builder().body(hbb).build();
+    	  
+    	for (int i = 0; i < seriesLen; i++) {
+    	    prices[i] = 25.0 + rng.nextDouble();
+    	    volumes[i] = rng.nextInt(10000);
+    	    times[i] = startDT.plusMinutes(i);
+    	}
+    	  
+    	DateTimeColumn x = DateTimeColumn.create("time", times);
+    	DoubleColumn y2 = DoubleColumn.create("price", prices);
+    	IntColumn y1 = IntColumn.create("volume", volumes);
+    	  
+    	ScatterTrace trace0 = ScatterTrace.builder( x, y2 )
```

Comment:
it looks like there's a mix of tabs and spaces here. please remove all tabs in this PR